vendor.conf

```diff
@@ -31,29 +31,29 @@ github.com/Microsoft/go-winio                       v0.4.14
 github.com/Microsoft/hcsshim                        v0.8.9
 github.com/opencontainers/go-digest                 v1.0.0
 github.com/opencontainers/image-spec                v1.0.1
-github.com/opencontainers/runc                      v1.0.0-rc10
-github.com/opencontainers/runtime-spec              v1.0.2
+github.com/opencontainers/runc                      v1.0.0-rc91
+github.com/opencontainers/runtime-spec              237cc4f519e2e8f9b235bacccfa8ef5a84df2875
 github.com/pkg/errors                               v0.9.1
 github.com/prometheus/client_golang                 v1.3.0
 github.com/prometheus/client_model                  v0.1.0
 github.com/prometheus/common                        v0.7.0
 github.com/prometheus/procfs                        v0.0.8
-github.com/russross/blackfriday                     v1.5.2
```

Comment:
probably needs blackfriday/v2 ? Perhaps I should dust off https://github.com/containerd/containerd/pull/4239, and change it to urfave v1.22.1, as I think there were some other changes needed

Reply:
I updated https://github.com/containerd/containerd/pull/4239

Reply:
yep let me add this back on

Reply:
https://github.com/containerd/containerd/pull/4239 was merged, so if you rebase, these changes would no longer be needed in this PR